manifests/templates/registry-host.yaml.in

```diff
@@ -78,11 +78,7 @@ spec:
         image: {{ .DockerRepo }}/cdi-func-test-registry-populate:{{ .DockerTag }}
         imagePullPolicy: {{ .PullPolicy }}
         securityContext:
-          capabilities:
-            add:
-            # building using buildah requires a properly installed shadow-utils package (which in turn requires SETFCAP)
-            # https://www.redhat.com/sysadmin/podman-inside-kubernetes
-            - SETFCAP
+          privileged: true
```

Comment:
privileged=true is a really large hammer, would prefer to not go to that extent.. I can help debug this error though! what were you seeing? this has been working for us for a long time now

Reply:
`registry-populate` in `cdi-docker-registry-host` failed to start, and here is the log:
```
kubectl logs -n cdi                    cdi-docker-registry-host-7587f89474-fk4gq registry-populate
health
building image tinycoreiso
Error during unshare(CLONE_NEWUSER): Permission denied
time="2025-01-03T05:32:59Z" level=error msg="parsing PID \"\": strconv.Atoi: parsing \"\": invalid syntax"
time="2025-01-03T05:32:59Z" level=error msg="(Unable to determine exit status)"
Exiting on error
```
After set privileged=true, the pod can start successfully.

Reply:
`cdi-docker-registry-host` starts successfully when the "SYS_ADMIN" capability is added. This might be because the Kubernetes cluster is being run as the root user. Since the Arm64 CI lane also runs as the root user, would it be acceptable to add the "SYS_ADMIN" capability in this case?
```
        securityContext:
          capabilities:
            add:
            - SETFCAP
            - SYS_ADMIN
```

Reply:
I'm fine with that

Reply:
Could you please just add it to https://github.com/kubevirt/containerized-data-importer/blob/54a278a3608cc0f4964cd5e4c0f9e362d11c33d1/manifests/templates/cdi-testing-scc.yaml.in#L14-L15 for our OpenShift folks to continue running normally

Reply:
Oh I think I see why CI fails. It appears you also need `SYS_CHROOT`. I tested that locally and it seems to work, but, I am a bit concerned about the extra permissions; maybe just apply it conditionally `if =~ kind` via a patch?

Reply:
Good suggestion!